duckduckgo/macos-browser
# Release 1.47.1

Task/Issue URL: https://app.asana.com/0/1199230911884351/1205002620586950/f
Tech Design URL:
CC: @shakyShane @jonathanKingston 

**Description**:

This PR is the release branch for 1.47.1, which bumps BSK to include Duck Player fixes.

The BSK version used in this PR is a hotfix release version based off of 64.0.0, which has been updated to point to C-S-S [4.25.0](https://github.com/duckduckgo/content-scope-scripts/releases/tag/4.25.0).

Note that this PR may be hard to review, as Duck Player is partially disabled at the time of opening. However, Tim has requested that we re-enable it, which should be done by Friday morning or so.

⚠️ I haven't opened a BSK PR as I would be handling the merge process for it by hand, but you can view the diff here: https://github.com/duckduckgo/BrowserServicesKit/compare/64.0.0...release/64.0.0.1

**Steps to test this PR**:
1. Run this branch, and make sure that Duck Player works as expected (assuming that Duck Player has been re-enabled)
2. Make sure that you can sign into app.plex.tv with a Google account



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
